sklearn/datasets/_twenty_newsgroups.py

```diff
@@ -239,6 +239,11 @@ def fetch_20newsgroups(
             The names of target classes.
 
     (data, target) : tuple if `return_X_y=True`
+        A tuple of two ndarrays. The first contains a 2D array of shape
+        (n_samples, n_classes) with each row representing one sample and each
+        column representing the features. The second array of shape
+        (n_samples,) contains the target samples.
```

Comment:
I made a small push to be sure that there are no trailing spaces and that we respect the PEP8/black convention (meaning no more than 88 characters per line).

You can check the commit that I just pushed. It should be enough for this PR. I will merge once the CI is green..